GuildedRoseLLC/Details/DetailsViewController.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 public class DetailViewController: UIViewController {
   
+    @IBOutlet public var quality: UILabel!
```

Comment:
Comment about naming. Right now, you have `quality`, `sellIn`, and `itemName`. This is a bit inconsistent, since one has the prefix of `item`, and the other two do not.

Additionally, when just looking at the variable name, there's nothing to differentiate `quality` the `UILabel` from `quality` the `Int`. I would suggest naming it `qualityLabel` or `itemQualityLabel`, so there is no ambiguity.